src/node.rs

```diff
@@ -49,18 +49,19 @@ use zksync_core::api_server::web3::backend_jsonrpc::{
     error::into_jsrpc_error, namespaces::eth::EthNamespaceT,
 };
 use zksync_state::{ReadStorage, StoragePtr, StorageView, WriteStorage};
-use zksync_types::vm_trace::Call;
 use zksync_types::{
     api::{Block, DebugCall, Log, TransactionReceipt, TransactionVariant},
     block::legacy_miniblock_hash,
     fee::Fee,
     get_code_key, get_nonce_key,
     l2::L2Tx,
+    l2::TransactionType,
     transaction_request::TransactionRequest,
     utils::{
         decompose_full_nonce, nonces_to_full_nonce, storage_key_for_eth_balance,
         storage_key_for_standard_token_balance,
     },
+    vm_trace::Call,
```

Comment:
Where is this used?

Reply:
Various places. This `use` statement was just combined. Wasn't sure why it was separated in the first place